test/controllers/application_controller_test.rb

```diff
@@ -188,11 +188,21 @@ class ApplicationControllerTest < ActionController::TestCase
     assert_equal 'private, no-store', response.headers['Cache-Control']
   end
 
+  test 'redirect to blocked license' do
+    skip unless ENABLE_PUBLIC_REGISTRATION
+
+    login_admin
+
+    @controller.send :redirect_to_license_blocked
+
+    assert_redirected_to license_blocked_url
+  end
+
   private
 
   def login_admin
-    User.find(users(:administrator).id).update_attribute(:logged_in, true)
+    User.find(users(:administrator).id).update_attribute :logged_in, true
```

Comment:
```suggestion
    users(:administrator).update_attribute :logged_in, true
```

Solo aprovechando que lo cambiaste =)

Reply:
_me la hizo_